pages/[league]/player/[id].tsx

```diff
@@ -0,0 +1,337 @@
+            setPlayerName(goalieStats[0].name);
+            setPlayerPosition('G');
+            setPlayerTeam(goalieStats[0].team.toString());
+            setPlayerHeight(goalieInfo[0].height.toString());
+            setPlayerWeight(goalieInfo[0].weight.toString());
+          }
+        }
+      }
+    }
+  });
+
+  const onSeasonTypeSelect = async (seasonType: SeasonType) => {
+    setFilterSeasonType(seasonType);
+  };
+
+  return (
+    <React.Fragment>
+      <NextSeo
+        title="Player"
+        openGraph={{
+          title: 'Player',
+        }}
+      />
+      <Header league={league} activePage="players" isSticky={false} />
+      <Container>
+        {isLoading && !playerError && (
+          <CenteredContent>
+            <PulseLoader size={15} />
+          </CenteredContent>
+        )}
+        <Main>
+          {playerError && (
+            <ErrorBlock>
+              Failed to load player info. Please reload the page to try again.
+            </ErrorBlock>
+          )}
+          <Table>
+            <TableRow>
+              <TableData>
+                <Button onClick={() => router.back()} inverse>
+                  <ButtonContent>
+                    Go Back
+                  </ButtonContent>
+                </Button>
+              </TableData>
+              <TableData>
+                <SelectorWrapper>
+                  <SeasonTypeSelector onChange={onSeasonTypeSelect} />
+                </SelectorWrapper>
+              </TableData>
+            </TableRow>
+          </Table>
+          {!isLoading && (
+            <>
+              <CenteredContent>
+                <PlayerInfo>{playerName} | {playerPosition} | {playerHeight} in | {playerWeight} lbs | {playerTeam}</PlayerInfo>
+              </CenteredContent>
+              {!isSkater && (
+                <>
+                  <TableHeading>Stats</TableHeading>
+                  <TableWrapper>
+                    <TableContainer>
+                      {goalieStats && (
+                        <SingleGoalieScoreTable data={goalieStats} />
+                      )}
+                    </TableContainer>
+                  </TableWrapper>
+                </>
+              )}
+              {isSkater && (
+                <>
+                  <TableHeading>Stats</TableHeading>
+                  <TableWrapper>
+                    <TableContainer>
+                      {skaterStats && (
+                        <SingleSkaterScoreTable data={skaterStats} />
+                      )}
+                    </TableContainer>
+                  </TableWrapper>
+                  <TableHeading>Advanced Stats</TableHeading>
+                  <TableWrapper>
+                    <TableContainer>
+                      {skaterStats && (
+                        <SingleSkaterAdvStatsTable data={skaterStats} />
+                      )}
+                    </TableContainer>
+                  </TableWrapper>
+                </>
+              )}
```

Comment:
I think we can make this a ternary since it is a simple true/false condition

Reply:
Also what is the table doing here? The only table should be that component

Reply:
i made this a ternary

Reply:
as for the additional table tags, those are present in the players page as well.  They are needed to make sure the pagination is lined up.  We should go back and look at that later.

Reply:
I must have missed that before. There is probably a much better way of doing this and i want to stay away from table tags for things outside of tables

Reply:
unless these are just poorly named divs that i set a bad convention for early on

Reply:
yea, I do not know enough about them to really understand why they are there.  I think we should open an issue on it and address it later as it is used throughout.